test.txt

```diff
@@ -0,0 +1 @@
+testing
```

Comment:
we'll want to remove this. Could you also add a pytest like we do here: https://github.com/msk-access/postprocessing_variant_calls/blob/main/tests/test_maf.py? Just testing if the command `--help` works is fine. No need to use real data.